data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -2074,12 +2075,12 @@ Robe of Useful Items 2																												TYPE:Magic.Wondrous.Robe.Conta
+Book of Vile Darkness																												TYPE:Magic.Artifact.Minor.Book																	COST:0	WT:3																																				SOURCEPAGE:p.277							ABILITY:Special Ability|AUTOMATIC|Negative Levels																																																															BONUS:VAR|NegLevels|1|!PRECLASS:1,SPELLCASTER=1																									SPROP:This is a work of ineffable evil-meat and drink to divine spellcasters of that alignment (LE, NE, CE). To fully consume the contents requires one week of study. Once this has been accomplished, an evil spellcaster gains a +1 inherent bonus to Wisdom and one experience level-receiving enough XP to put the character's XP total midway between the minimum needed for his new (higher) level and the minimum needed for the level beyond that. All lawful neutral, neutral, or chaotic neutral characters who touch the book take 5d4 points of damage, and reading its pages causes them to become evil (Will DC 13 negates). Such converts immediately seek out an evil cleric to confirm their new alignment (with an atonement spell). Divine spellcasters neither good nor evil (LN, N, CN) who read the book either lose 2d6*1,000 XP (01-50 on d%) or become evil without benefit from the book (51-100). Good divine spellcasters (LG, NG, CG) perusing the pages of the book of vile darkness have to make a DC 16 Fortitude save or die. If they do not die, they must succeed on a DC 15 Will save or suffer from a continuous confusion effect (per the insanity spell). In the latter event, even if the save is successful, the character loses 20,000 XP, minus 1,000 for each point of Wisdom he has. (This calculation cannot result in an XP gain.) Other characters of good alignment take 5d6 points of damage from just handling the tome. If such a character looks inside, there is an 80% chance that an evil outsider attacks the character that night. Strong evocation [evil]; CL 19th
```

Comment:
Looks like this one was already missing an escape %. for the d% line, and escape % removed.

Reply:
SPROP not working as documented

Do you actually see the text output appearing correctly with % | VAR anywhere?